### PR TITLE
Reworked elemental doctrines

### PR DESCRIPTION
**Pull request by MemeWizard**
## Changelog:
- Reworked spirit and decay magical doctrines for future usage:
  -  It's needed for future branches, so characters can potentially bully water elementals while having a good relationship with fire elementals, etc.
  - Removed 2 old (spirit&decay) doctrines
  - Reworked 2 and added 3 new sub-doctrines for each of 4 elements doctrines
 - Changed a few starting doctrines for various faiths

## Dev changelog:
- Reworked a few triggers, replaced several triggers with one.

## Tests:
- [x] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [x] The mod takes less than 5.5 GB in the Task Manager (Windows)

# How to test:
- Make sure that tooltips correctly inform you why this specific doctrine can not be picked when your character reforms or create the new faith.
- Tell me if any of the elemental doctrines are assigned wrongly for a faith. Some clarifications:
  - Controlled  means that mages of that faith are allowed to use this element
  - Harmonised means that this element is allowed to be communed with by shamans of that faith, and Idolised-Spirit means that the faith promotes shamans to commune with this element
  - Enslaved means that this element is allowed to be enslaved by dark shamans of that faith, and Idolised-Decay means that the faith promotes dark shamans to enslave this element

### CODE REVIEW

File: common/religion/religions/loa_group.txt [hunk 1]
```diff
@@ -531,7 +535,14 @@
...
 			doctrine = tenet_esotericism
 			doctrine = tenet_warmonger
-			
+
+			#Magic
+			doctrine = doctrine_death_magic_approved
+			doctrine = doctrine_elemental_fire_spirit_magic_accepted
+			doctrine = doctrine_elemental_water_spirit_magic_accepted
+			doctrine = doctrine_elemental_air_spirit_magic_accepted
+			doctrine = doctrine_elemental_earth_spirit_magic_accepted
```
Comment (zumbak04) [inline common/religion/religions/loa_group.txt:544]: Why Hakkari aren't enslaving them?

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/religion/religions/loa_group.txt (hunk 1 above)
Comment (MemeWizard) [reply common/religion/religions/loa_group.txt]: Should be fixed in 5ab32eb

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Review by riioze — APPROVED** [review]
The 1st point is good but I'm sorry I didnt found the Elemental doctrines

**Review by arithon — APPROVED** [review]
loc LGTM (will check german later ingame, but most likely in the LC)

**Review by TheGroxEmpire — COMMENTED** [review]
- This tooltip is confusing to me. The part where it mentions two syncretism Tenets. There is a grammatical error on how the or is used here but I can't suggest changes before I understand what it meant:
<details>

image

</details>

**Review by TheGroxEmpire — APPROVED** [review]
Everything else look good to me!

**Comment by MemeWizard** [comment]
@zumbak04 @arithon @TheGroxEmpire  @riioze thanks for reviews <3

**Pull request merged**